corsheaders/checks.py

```diff
@@ -0,0 +1,88 @@
+import collections
+import numbers
+
+from django.core import checks
+from .conf import corsheaders_settings
+
+@checks.register
+def check_settings(app_configs, **kwargs):
+
+    errors = []
+    if not isinstance(corsheaders_settings.CORS_ALLOW_HEADERS, collections.Sequence):
+        errors.append(
+            checks.Warning(
+                "CORS_ALLOW_HEADERS must be sequence",
+                id="django-cors-headers.W001"
```

Comment:
I think we should use `corsheaders` in our ID's, because that's the module name. It is indeed a bit annoying that the PyPI name and the module name are different.